src/components/pages/navbar/NavbarContainer.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
Overall this is an awesome refactoring of the Nav! I like what you've done to give us flexibility to render normal links or routes depending on the application needs. That's thinking ahead for the future for sure. Well done.